<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-LU09BBF" ignored="0" name="Test" tests="10" failures="2" timestamp="26 Apr 2021 04:54:26 GMT" time="0.14" errors="0">
  <testcase name="tc1" time="0.015" classname="testNG.TestNgRerunFaildTestCases">
    <skipped/>
  </testcase> <!-- tc1 -->
  <testcase name="tc1" time="0.0" classname="testNG.TestNgRerunFaildTestCases">
    <skipped/>
  </testcase> <!-- tc1 -->
  <testcase name="tc1" time="0.0" classname="testNG.TestNgRerunFaildTestCases">
    <skipped/>
  </testcase> <!-- tc1 -->
  <testcase name="tc1" time="0.0" classname="testNG.TestNgRerunFaildTestCases">
    <skipped/>
  </testcase> <!-- tc1 -->
  <testcase name="tc1" time="0.0" classname="testNG.TestNgRerunFaildTestCases">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testNG.TestNgRerunFaildTestCases.tc1(TestNgRerunFaildTestCases.java:9)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- tc1 -->
  <testcase name="tc2" time="0.0" classname="testNG.TestNgRerunFaildTestCases">
    <skipped/>
  </testcase> <!-- tc2 -->
  <testcase name="tc2" time="0.0" classname="testNG.TestNgRerunFaildTestCases">
    <skipped/>
  </testcase> <!-- tc2 -->
  <testcase name="tc2" time="0.0" classname="testNG.TestNgRerunFaildTestCases">
    <skipped/>
  </testcase> <!-- tc2 -->
  <testcase name="tc2" time="0.0" classname="testNG.TestNgRerunFaildTestCases">
    <skipped/>
  </testcase> <!-- tc2 -->
  <testcase name="tc2" time="0.0" classname="testNG.TestNgRerunFaildTestCases">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testNG.TestNgRerunFaildTestCases.tc2(TestNgRerunFaildTestCases.java:15)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- tc2 -->
</testsuite> <!-- Test -->
